### ISSUE TITLE
Update problem

### ISSUE THREAD

**Issue by onebeat** [tine20/tine20#7014]
I wanted to upload new version of tine but i get this error...
![Untitled](https://user-images.githubusercontent.com/4642271/54239508-a4090980-451b-11e9-8068-b2eff5c8f71d.jpg)

**Comment by ignoreigor** [comment]
Hi,
try an update via console with  "php setup.php --update", use sudo if necessary.

For me, in this version, I had to reduce my config temporarily to a minimal config (only db, without cache, session e.g.) to get an successful update. Otherwise even the update hangs when started in CLI.

**Comment by lab-at-nohl** [comment]
Please enable log-file in config.inc.php and post information at level debug here...

**Comment by onebeat** [comment]
Hello, i have tried your way but no luck...here is my debug log from setup(upgrade) script
[error-log.txt](https://github.com/tine20/tine20/files/2962508/error-log.txt)

**Comment by lab-at-nohl** [comment]
```Tinebase_Exception_Backend -> mysql >= 5.7.5 | mariadb >= 10.0.5 required```

Your DB is too old. Unfortunatly without stating it in a more prominent way the release 2019.02 drastically increased the requirements - MySQL/MariaDB and PHP 7.x.

What is your system?

**Comment by onebeat** [comment]
I am runing on linux with php 7.2 version and MySQL Version 5.6.41

So i should upgrade to new version right ?

**Comment by lab-at-nohl** [comment]
Yes, you need to upgrade your MySQL obviously. Which distro are you on? I had chosen MariaDB over MySQL from the beginning therefore I could use their repositiory - worked flawlessly on CentOS 7.

**Comment by onebeat** [comment]
I am using this on shared hosting so i can't see what is distro but my guess is centos with cloudlinux...

**Comment by lab-at-nohl** [comment]
Have you realized that the latest release lowered its requirements on DBs again? Maybe you can install 2019.02.4?
